compose-profiles/docker-compose-definitions.yml

```diff
@@ -124,6 +124,10 @@ services:
     environment:
       - LOG_LEVEL=3
       - UNLYNX_GROUP_FILE_PATH=/medco-configuration/group.toml
+      - UNLYNX_TIMEOUT_SECONDS=180
+      - I2B2_WAIT_TIME_SECONDS=180
+      - MEDCO_OBFUSCATION_MIN=5
+      - GA_DB_PORT=5432
```

Comment:
Why adding those variables here, in the medco-cli-client? They shouldn't need it

Reply:
You're right. I added them just to get rid of the warning messages that appear when you use the medco cli which complains that they are not defined. They are not strictly needed, so if you prefer I can remove them.

Reply:
Yeah it's better to remove them, it's confusing.

I am aware for the annoying warning messages, and this should be addressed properly at some point.